NEWS

```diff
@@ -48,6 +48,12 @@ CHANGES WITH 253 in spe:
         * systemd-fstab-generator now supports x-systemd.makefs option for
           /sysroot (in the initrd).
 
+        * New Network param [DHCPv4]SocketPriority= to assign the Linux socket priority
```

Comment:
`New Network param [DHCPv4]SocketPriority=` -> `The [DHCPv4] section in .network file gained new SocketPriority= setting that assigns the Linux socket priority...`